astro/src/content/docs/apis/identity-providers/_hypr-request-body.mdx

```diff
@@ -38,14 +38,21 @@ import JSON from 'src/components/JSON.astro';
   <IdentityProviderLinkingStrategyRequestParameter idp_since={props.idp_since} idp_linking_strategy={props.idp_linking_strategy} idp_display_name={props.idp_display_name}/>
 
   <APIField name="identityProvider.name" type="String" optional since="1.61.0">
-    The display name of this provider instance. Required when using a provided <InlineField>identityProviderId</InlineField>.
+    The display name of this provider instance. Required when using a provided <InlineField>identityProviderId</InlineField> or <InlineField>identityProvider.tenantId</InlineField>.
   </APIField>
   <APIField name="identityProvider.relyingPartyApplicationId" type="String" required>
     The Relying Party Application identifier provided to you by HYPR.
   </APIField>
   <APIField name="identityProvider.relyingPartyURL" type="String" required>
     The Relying Party URL provided to you by HYPR.
   </APIField>
+  <APIField name="identityProvider.tenantId" type="UUID" optional since="1.62.0">
+    The unique Id of the Tenant.
+
+    Providing a value creates an identity provider scoped to the specified tenant, otherwise a global identity provider is created. Tenant-scoped identity providers can only be used to authenticate in the context of the specified tenant while global identity providers can be used with any tenant.
```

Comment:
```suggestion
    The unique ID of the Tenant.

    Providing a value creates an identity provider scoped to the specified Tenant, otherwise a global identity provider is created. Tenant-scoped identity providers can only be used to authenticate in the context of the specified tenant. Global identity providers can be used with any tenant.
```
same as above